.eslintrc.json

```diff
@@ -22,8 +22,17 @@
     "ecmaVersion": 2018,
     "sourceType": "module"
   },
-  "plugins": ["react", "jsx-a11y", "import", "jest"],
```

Comment:
this plugins are provided out of the box by extending next or next/core-web-vitals